elodie.py

```diff
@@ -110,7 +110,7 @@ def _import(destination, source, file, album_from_folder, trash, allow_duplicate
         else:
             files.add(path)
 
-    for current_file in files:
+    for current_file in sorted(files):
```

Comment:
Can merge this if we switch to using the sort method on `files`. Sorted makes a copy which takes up additional memory while `.sort()` is in-place and seems okay in this case.